settings.gradle

```diff
@@ -25,4 +25,5 @@ include 'reactivesocket-spectator'
 include 'reactivesocket-test'
 include 'reactivesocket-transport-aeron'
 include 'reactivesocket-transport-local'
-include 'reactivesocket-transport-tcp'
\ No newline at end of file
+include 'reactivesocket-transport-tcp'
+include 'reactivesocket-tck-drivers'
```

Comment:
You are right. No tests should be running.  I have removed the tests tasks from the old gradle file.  https://github.com/ReactiveSocket/reactivesocket-java/pull/249/files#diff-516721e9b05fb6098ef05dbb7722adc3